src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1891,4 +1897,67 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addUidSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('uid')
+                    ->info('Uid configuration')
+                    ->{class_exists(AbstractUid::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                    ->children()
+                        ->arrayNode('uuid_factory')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->enumNode('default_named_version')
+                                    ->defaultValue(UuidV5::class)
+                                    ->cannotBeEmpty()
+                                    ->values([UuidV5::class, UuidV3::class])
+                                        ->beforeNormalization()
+                                        ->always()
+                                        ->then(static function ($value) {
+                                            if ('v5' === $value) {
```

Comment:
cannot happen not - or you need to list them in the enum (which would be more friendly to yaml configs)